python/bigipconfigdriver.py

```diff
@@ -596,67 +846,83 @@ def _do_reset(self):
                         self.cleanup_backoff()
                     break
 
-                try:
-                    start_time = time.time()
-
-                    config = _parse_config(self._config_file)
-                    if 'resources' not in config:
-                        continue
-                    verify_interval, _ = _handle_global_config(config)
-                    _handle_openshift_sdn_config(config)
-                    self.set_interval_timer(verify_interval)
-
-                    # Manually create custom profiles;CCCL doesn't yet do this
-                    if 'customProfiles' in config['resources']:
-                        for profile in config['resources']['customProfiles']:
-                            _create_client_ssl_profile(self._bigip, profile)
-                            customProfiles = True
-
-                    cfg = create_config_kubernetes(self._bigip, config)
-                    if self._bigip.regenerate_config_f5(cfg):
-                        # Error occurred, perform retries
-                        log.warning(
-                            'regenerate operation failed, restarting')
-                        self.handle_backoff()
-                    else:
-                        if (self._interval and self._interval.is_running() is
-                                False):
-                            self._interval.start()
-                        self._backoff_time = 1
-                        if self._backoff_timer is not None:
-                            self.cleanup_backoff()
+                start_time = time.time()
+
+                config = _parse_config(self._config_file)
+                if 'resources' not in config:
```

Comment:
If you have no resources, but do have an Openshift SDN config (config['openshift-sdn']), I think you do want to configure.

As well, if you have no resources, but are supposed to be in charge of 2 partitions, I think you do want to "configure" them (to remove everything from inside of them).

If that's right, I don't think you want this "continue"